src/App.js

```diff
@@ -56,6 +57,7 @@ function App() {
           <Route path="/social" element={<PilotWall onLogin={()=>{onLogin(false)}} onLogout={onLogout} />} />
           <Route path="/follows" element={<FollowsWall />} />
           <Route path="/gameconfigurator" element={<GameConfigurator />} />
+          <Route path="/gameviewer" element={<GameViewer gid={queryParams.get && queryParams.get('gid')} />} />
```

Comment:
check if user not logged and this kind tests

Reply:
following the Landing route works correctly
Needs to be checked for not auth users directly to the gam viewer call